docs/MachineConfigDaemon.md

```diff
@@ -129,3 +131,7 @@ The draining of pods on the only master node will not evict the control plane as
 ### Node drain etcd static pods on masters
 
 Etcd is co-located on master nodes as static pods. The draining behavior defined above prevents draining of static pods to prevent interference to etcd cluster by the daemon.
+
+## Annotating on SSH access
+
+RHCOS nodes in Openshift are not meant to be manually accessed via SSH. MCD uses logind to watch for login sessions, which, upon detection, warns the user and annotates the node with `machineconfiguration.openshift.io/rhcosSSH=accessed`. This in turn will be used to warn cluster admins.
```

Comment:
At some point I think we're going to need to link to a more canonical doc on this as we're...somewhat inconsistent in our "is SSH allowed/encouraged" phrasing.  But this is OK for now.

Reply:
Agreed